clients/index.js

```diff
@@ -417,3 +418,17 @@ ApplicationClients.prototype.updateKValues = function updateKValues() {
         self.serviceProxy.updateServiceChannels();
     }
 };
+
+ApplicationClients.prototype.updateKillSwitches = function updateKillSwitches() {
+    var self = this;
+    self.serviceProxy.unblockAll();
+    var killSwtiches = self.remoteConfig.get('killSwitch', []);
+
+    for (var i = 0; i < killSwtiches.length; i++) {
+        var value = killSwtiches[i];
+        var edge = value.split('~~');
+        if (edge.length === 2 && (edge[0] !== '*' || edge[1] !== '*')) {
```

Comment:
It wasn't supported in admin tool neither. I guess it doesn't make sense to kill everything.